tests/test-books-api.js

```diff
@@ -0,0 +1,178 @@
+require('dotenv').config()
+const dbbooks = require('../dbbooks')
+const assert = require('assert')
+const axios = require('axios')
+const dbusers = require('../dbusers')
+
+const State = {
+    SUCCESS: "Success",
+    ERROR: "Error"
+}
+const ErrorResponseMessage = "Error occurs when execute query on database"
+
+const isbn = '123456sdfa123'
+const name = 'Bleach'
+const author = 'Tite Kubo'
+const publisher = 'Kim Dong'
+const quantity = 145
+const id_category = '13fbb02e-9d06-48de-9ff4-276ff5959e1f'
```

Comment:
Should not use a specific Id in testing because we can't assume the database has already haven it. Add new entity if you want to use it.

Reply:
I fixed it